Bug t radiative

Sumit112192 opened the pull request:
### :pencil: Description

**Type:** :beetle: `bugfix`| memo: `documentation`
![image](https://github.com/tardis-sn/tardis/assets/53135486/8582b2a7-d917-4ed0-ba9b-62135964703a)
ref. https://tardis-sn.github.io/tardis/physics/setup/model.html#Temperatures

https://github.com/tardis-sn/tardis/blob/1718002b4db7fd1a93e475844582ed9b41aa7cdf/tardis/model/parse_input.py#L726-L731
The code doesn't match with the documentation. So, I have changed it accordingly.
I have directly used `packet_source.temperature` rather than first computing `lambda_wien_inner`



Slight documentation typos: 
The documentation string of `parse_structure_config` was repeated twice, so I removed the extra one.
Also, there were some instances where `v_inner, v_outer` were labeled as `r_inner, r_outer`. So, I have changed them as well.


Also, If we pass `packet_source` as one of the function arguments, why define it again in the body?
https://github.com/tardis-sn/tardis/blob/1718002b4db7fd1a93e475844582ed9b41aa7cdf/tardis/model/parse_input.py#L589-L618
### :pushpin: Resources

Examples, notebooks, and links to useful references.


### :vertical_traffic_light: Testing

How did you test these changes?

- [ ] Testing pipeline
- [ ] Other method (describe)
- [ ] My changes can't be tested (explain why)


### :ballot_box_with_check: Checklist

- [x] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

> **Note:** If you are not allowed to perform any of these actions, ping (@) a contributor.
@andrewfullard @jamesgillanders Please review this pull request.

andrewfullard commented on tardis/model/parse_input.py:
```diff
@@ -723,10 +687,8 @@ def calculate_t_radiative_from_t_inner(geometry, packet_source):
-        lambda_wien_inner
-        * (1 + (geometry.v_middle - geometry.v_inner_boundary) / const.c)
+    t_radiative = packet_source.temperature / (
+        (1 - (geometry.v_middle - geometry.v_inner_boundary) / const.c)
```
The documentation is incorrect here: the `1 - ` should be `1 + ` because temperature should *decrease* further out. The other change (packet_source.temperature) is fine.

andrewfullard replied:
You can modify the documentation to fix the sign of the function

Sumit112192 replied:
Yes !!! That makes sense. I will change the documentation and will redo this change to the original.

DeerWhale commented:
Thanks for spotting the duplication of the packet initialization, I have opened an issue #2585 and our team members will look into this further. Feel free to dive deeper if you are interested as well!

Sumit112192 commented:
@andrewfullard I have made the necessary changes.

The changes that I have made till now.
1. Updated the documentation so that doppler factor is $1+\frac{\Delta v}{v}$ instead of $1-\frac{\Delta v}{v}$
2. Fixed some common typos in the documentation strings of the functions
3. Change suggested by Black in the file `tardis/model/parse_input.py`

Sumit112192 commented:
I don't know why Black's test is failing on `tardis/model/parse_input.py`, but I have made the change that Black suggested.

![image](https://github.com/tardis-sn/tardis/assets/53135486/8d9edf8d-43f3-4042-903a-373866029595)

Sumit112192 commented:
It seems that the suggestions by black installed through `sudo apt install black` on Ubuntu are different from those of black installed through `pip install black`.

Sumit112192 commented:
Sorry for all those commits. I am still trying to figure out how branching works in Git, and hopefully, I will not repeat the same mistake.

andrewfullard commented:
> It seems that the suggestions by black installed through `sudo apt install black` on Ubuntu are different from those of black installed through `pip install black`.

Make sure you use the version of black that is installed with the TARDIS environment

DeerWhale commented on tardis/model/parse_input.py:
```diff
@@ -33,42 +33,6 @@ def parse_structure_config(config, time_explosion, enable_homology=True):
-    """
-    """
-    Parse the structure configuration data.
-
```
Is this deletion of the doc strings intended?

Sumit112192 replied:
They were repeated twice.

DeerWhale replied:
Sorry my bad didn't notice it, thanks for spotting the duplicates!
